shared/src/main/scala/amf/client/remod/ErrorHandlerProvider.scala

```diff
@@ -3,7 +3,7 @@ package amf.client.remod
 import amf.client.parse.DefaultParserErrorHandler
 import amf.core.errorhandling.ErrorHandler
 
-trait ErrorHandlerProvider {
+private[amf] trait ErrorHandlerProvider {
```

Comment:
This shouldn't be private. @pope1838

Reply:
You are right. The adopter could register an ErrorHandlerProvider. Is a client interface